controllers/apply-next/simple/form-model.js

```diff
@@ -182,16 +182,7 @@ const sectionLegalContact = {
             fieldsets: [
                 {
                     legend: { en: 'Who is your legally responsible contact?', cy: '' },
-                    introduction: {
```

Comment:
Moved this help text inline as each paragraph really related to a discreet field.